lib/entry/FlavorDownloader.js

```diff
@@ -364,7 +365,64 @@ var FlavorDownloader = (function() {
         });
     }
 
-    FlavorDownloader.prototype.start = function (onIterationEndCallback) {
+    function getTotalBitrate(fallbackBitrate) {
+        if (this.encoderInformation && this.encoderInformation.Encoder) {
+            // Support all types of stream playing: Regular, video only and audio only
+            switch (checkStreamType.call(this)) {
```

Comment:
let totalBitrate = 0;
if (this.encoderInformation.Encoder.videoBitrate ) {
 totalBitrate+=this.encoderInformation.Encoder.videoBitrate ;
}
if (this.encoderInformation.Encoder. audioBitrate ) {
 totalBitrate+=this.encoderInformation.Encoder. audioBitrate ;
}